packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -19,22 +19,28 @@ import {
 } from '../constants';
 import { isValidOfflineFormEncounter } from '../offline-forms/offline-form-helpers';
 
-export function useFormEncounters(cachedOfflineFormsOnly = false, patientUuid: string = '', visitUuid: string = '') {
+function useCustomFormsUrl(patientUuid: string, visitUuid: string) {
   const { customFormsUrl, showHtmlFormEntryForms } = useConfig<ConfigObject>();
   const hasCustomFormsUrl = Boolean(customFormsUrl);
-  let url = hasCustomFormsUrl
-    ? customFormsUrl.indexOf('?') === -1
-      ? `${customFormsUrl}?patientUuid=${patientUuid}`
-      : customFormsUrl
-    : showHtmlFormEntryForms
-      ? formEncounterUrl
-      : formEncounterUrlPoc;
-  url = interpolateUrl(url, {
+
+  const baseUrl = hasCustomFormsUrl ? customFormsUrl : showHtmlFormEntryForms ? formEncounterUrl : formEncounterUrlPoc;
+
+  const url = interpolateUrl(baseUrl, {
     patientUuid: patientUuid,
     visitUuid: visitUuid,
     representation: customFormRepresentation,
   });
 
+  return {
+    url,
+    hasCustomFormsUrl,
+    showHtmlFormEntryForms,
```

Comment:
Why are we returning this?

Reply:
We need **hasCustomFormsUrl** to prevent filtering **customFormUrl results**. If we remove `formEncounterUrlPoc`, we can also eliminate the filter that excludes forms with "component" in their name (AMPATH use case). This would leave just one filter for published forms. I'm removing showHtmlFormEntryForms as it's unused.

Reply:
Sorry, I was talking very specifically about `showHtmlFormEntryForms` since we don't consume it in the other hook.